.github/workflows/tests.yml

```diff
@@ -246,20 +313,9 @@ jobs:
       matrix:
         # https://github.com/jetstack/cert-manager/tags
         path:
-        - samples/chartmuseum
-        - samples/core
-        - samples/database
         - samples/harbor
         - samples/harbor-full
-        - samples/jobservice
-        - samples/notary
-        - samples/notary-server
-        - samples/notary-signer
-        - samples/portal
-        - samples/registry
-        - samples/registryctl
```

Comment:
:-(

Reply:
there're too many check jobs to watch, so I'd like to do some simplify, especially these samples tests are already covered by `samples/harbor-full`